tripaldocker/Dockerfile-php8.2-pgsql13

```
@@ -178,7 +178,7 @@ RUN cd /var/www/drupal \
   --account-name=drupaladmin \
   --account-pass=some_admin_password \
   --site-mail="drupaladmin@localhost" \
-  --site-name="Tripal 4 on Drupal 10 DEVELOPMENT" \
+  --site-name="Tripal 4.x-dev on Drupal 10.1.0" \
```

Use the Drupal version arguement here so that it changes based on the version requested. This is because on line 3 you specify the default version of Drupal to use. However, when building the image you can pass in a build-arg to specify a different drupal version and we do not want a mismatch in the name :-) 
 
```suggestion
  --site-name="Tripal 4.x-dev on Drupal ${drupalversion}" \
```

---

Doh, thanks. Using 10.1.0 for our purposes I had rigid version in the brain. 😅